<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RigidTrackClass</class>
 <widget class="QMainWindow" name="RigidTrackClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RigidTrack</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnStartCamera">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Tracking</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStatus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>771</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZero">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Reference Point</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalibrate">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate
Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadCalib">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load
Calibration</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>681</y>
      <width>791</width>
      <height>171</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbHeadingOffset">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>136</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>136</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Heading Offset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leIPDrone">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>136</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.4.2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Adress of Drone Wifi Chip Press Return to Apply Changes</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopCamera">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Stop Tracking</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RigidTrack.qrc"/>
 </resources>
 <connections/>
</ui>
